DEPLOYMENT.md

```diff
@@ -18,21 +18,34 @@ gcloud config set project mfte-simple-92c08
 
 ## Deploying
 
-Make sure the app and docker containers are built:
```

Comment:
DNPTL: @mmcknett do you think it's a good idea to get rid of everything associated with Docker/"old way" of deploying? That's what I did in this PR but open feedback!

Reply:
What does DNPTL stand for? I think getting rid of docker & nginx now is the right thing to do!

Reply:
@mmcknett "does not pertain to line" Thanks for asking. People use it at work and I sometimes assume I'm the only one not knowing an acronym!